### PR TITLE
docs: Initial readme

### PR DESCRIPTION
**Pull request by twpol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+
+| Data | Type | Explanation |
+|---|---|---|
+| Category | Enum | Route, Train Car, Train Set |
+| Compensation model | Enum | Commercial, Donation, Free |
+| Name | Text | Single-line |
+| Description | Text | Multi-line, basic Markdown (bold/italic, lists) |
+| Author | Text | Single-line |
+| Image | ? | **What are we doing here???** |
+| Link | URL | This is where interested people will be directed, so it should feature a prominent download or purchase option |
```
Comment (twpol) [inline README.md:43]: These metadata fields might change when I apply the schema.org research onto the data files

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+- Train cars - *one or more related locomotives and/or wagons*
+- Train sets - *one or more related consists*
+
+## Dependencies
+
+Content in this collection **must not** require any separate or additional content to work, including the original Microsoft Train Simulator content. However, it is completely fine for included content to link to other content which **does** require separate or additional content to work.
+
+For example, if a route includes activities or timetables, all referenced train sets and train cars must also be included in a single download. If that is not possible, affected activities and timetables may be offered as a separate download, linked from the route's webpage, but only the route will be included in this collection.
+
+We understand that this is a tricky limitation for some content. However, we hope that content creators understand that we are currently focusing on ease of use, and we have plans to address this in the future.
```
Comment (cjakeman) [inline README.md:29]:
```suggestion
We understand that this limitation will exclude some popular content. However, we hope that content creators understand that we are currently focusing on ease of installation, and we have plans to address this limitation shortly.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+Content in this collection must have the following metadata:
+
+| Data | Type | Explanation |
+|---|---|---|
+| Category | Enum | Route, Train Car, Train Set |
+| Compensation | Enum | Commercial, Donation, Free |
+| Name | Text | Single-line |
+| Description | Text | Multi-line, basic Markdown (bold/italic, lists) |
+| Author | Text | Single-line |
+| Image | ? | **What are we doing here???** |
```
Comment (cjakeman) [inline README.md:42]:
```suggestion
| Image | ? | A small image or logo that identifies the content. Recommended width = 600 px |
```

**Review by cjakeman — COMMENTED** [review]
James,
My thoughts based on our chat yesterday.
Chris

**Review by cjakeman — APPROVED** [review]
All good.

Thanks, James.

Approved and merged.

**Pull request merged**